src/pt/models/typeAhead.ts

```diff
@@ -0,0 +1,31 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+
+@ObjectType()
+export class TypeAhead {
+    @Field({ nullable: true })
+    id: string;
+
+    @Field({ nullable: true })
+    score: number;
```

Comment:
Shouldn't we explicity add '() => Int' in all such fields because nestjs considers them float by default?

Reply:
@basuojha25